xds/src/main/java/io/grpc/xds/ServerXdsClient.java

```diff
@@ -70,11 +70,10 @@
       ScheduledExecutorService timeService,
       BackoffPolicy.Provider backoffPolicyProvider,
       Supplier<Stopwatch> stopwatchSupplier,
-      boolean useNewApiForListenerQuery,
       String instanceIp,
       String grpcServerResourceId) {
     super(channel, useProtocolV3, node, timeService, backoffPolicyProvider, stopwatchSupplier);
-    this.useNewApiForListenerQuery = useProtocolV3 && useNewApiForListenerQuery;
+    this.useNewApiForListenerQuery = useProtocolV3;
```

Comment:
Should `this.useNewApiForListenerQuery` potentially be renamed to `this.useProtocolV3`?

Reply:
Hmmm..  it's used to decide between the new Listener API vs old API so this seems more appropriate. In any case a lot of this code will get cleaned up/refactored when the old API support is removed as well as combining client + server code. So this member won't exist any more AFAICS